packages/grid/_modules_/grid/components/Pagination.tsx

```diff
@@ -64,7 +70,16 @@ export function Pagination() {
   return (
     // @ts-ignore TODO remove once upgraded v4 support is dropped
     <TablePagination
-      classes={classes}
+      classes={{
+        ...(isMuiV5()
```

Comment:
maybe it's worth moving in the scope above , create a small variable so it's more readable and a bit less verbose.

Reply:
Do you mean, that i would then look like this?

```suggestion
        ...(isMuiV5
```

Reply:
Yes I meant the whole classes to avoid {()} and extra lines...